src/test/java/com/uid2/shared/store/reader/RotatingKeysetProviderTest.java

```diff
@@ -139,16 +138,24 @@ public void loadContentMultipleEntries() throws Exception {
         Assert.assertEquals(6, count);
 
         // Site 2 can access 1, it can access its own, but not 3
-        Assert.assertTrue(canAccessKey(2, 1));
-        Assert.assertTrue(canAccessKey(2, 2));
-        Assert.assertFalse(canAccessKey(2, 3));
+        Assert.assertTrue(canAccessKey(2, 1, MissingAclMode.ALLOW_ALL));
+        Assert.assertTrue(canAccessKey(2, 1, MissingAclMode.DENY_ALL));
+        Assert.assertTrue(canAccessKey(2, 2, MissingAclMode.ALLOW_ALL));
+        Assert.assertTrue(canAccessKey(2, 2, MissingAclMode.DENY_ALL));
+        Assert.assertFalse(canAccessKey(2, 3, MissingAclMode.ALLOW_ALL));
+        Assert.assertFalse(canAccessKey(2, 3, MissingAclMode.DENY_ALL));
         //Only 4 can access null list
-        Assert.assertTrue(canAccessKey(4,4));
-        Assert.assertFalse(canAccessKey(2, 4));
+        Assert.assertTrue(canAccessKey(4,4, MissingAclMode.ALLOW_ALL));
+        Assert.assertTrue(canAccessKey(4,4, MissingAclMode.DENY_ALL));
+        Assert.assertTrue(canAccessKey(2, 4, MissingAclMode.ALLOW_ALL));
+        Assert.assertFalse(canAccessKey(2, 4, MissingAclMode.DENY_ALL));
         //Can still access if there is a duplicate
-        Assert.assertTrue(canAccessKey(1, 5));
+        Assert.assertTrue(canAccessKey(1, 5, MissingAclMode.ALLOW_ALL));
```

Comment:
should also ensure duplicates don't mess with denied sites. eg test for "canAccessKey(4, 5)"